scripts/ci.sh

```diff
@@ -45,7 +45,7 @@ verifyMarkdown() {
 
 npm ci
 npm run typecheck & npm run lint
-npm run test
+npm query .workspace | jq -r '.[].location' | sed -e 's,.*/,,' | xargs -P 0 -L1 npm run test -w
```

Comment:
We might be able to avoid having to define the empty test scripts by using the `--if-present` flag.

```suggestion
npm query .workspace | jq -r '.[].location' | sed -e 's,.*/,,' | xargs -P 0 -L1 npm run test --if-present -w
```